src/main/java/com/example/cosmocats/domain/Cart.java

```diff
@@ -0,0 +1,12 @@
+package com.example.cosmocats.domain;
+
+import lombok.Data;
+
+import java.util.List;
+import java.util.UUID;
+
+@Data
```

Comment:
I would prefer you to use the @Value here, because if we could do sth idempotent we should